.github/workflows/s3-sync-dev.yaml

```diff
@@ -0,0 +1,23 @@
+name: Upload Cloud Formation - Dev
+
+on:
+  push:
```

Comment:
this causes the cfn template to update on all pushes to any branch that isn't stable, which I think is probably not what we want, particularly if there are multiple PRs being worked on in parallel.

Reply:
This triggers on non-stable branch pushes, but it uses a different path for DEST_DIR ("deployment/dev" instead of "deployment/latest"), which gives us a way to test dev branches in CloudFormation before they are merged into stable. Regarding the parallel dev branches, we could include the branch name in the DEST_DIR path so each branch was uploaded to a unique location.